<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaterRecClass</class>
 <widget class="QWidget" name="PaterRecClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>615</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>615</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PaterRec</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>617</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>7</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item row="1" column="0">
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCommandLinkButton" name="openFilebtn">
          <property name="text">
           <string>选择图片</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>1</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>1</number>
          </property>
          <item row="1" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="rightMargin">
             <number>2</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="bottomMargin">
               <number>1</number>
              </property>
              <item>
               <widget class="QCheckBox" name="tribtn">
                <property name="text">
                 <string>三角形[蓝]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="rectbtn">
                <property name="text">
                 <string>四边形[红]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cirbtn">
                <property name="text">
                 <string>圆形[紫]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="polybtn">
                <property name="text">
                 <string>多边行[绿]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="topMargin">
               <number>1</number>
              </property>
              <item>
               <widget class="QPushButton" name="appbtn">
                <property name="text">
                 <string>应用</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="resetbtn">
                <property name="text">
                 <string>重选</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="imgShowWindow">
            <property name="minimumSize">
             <size>
              <width>480</width>
              <height>480</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>960</width>
              <height>960</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>480</width>
              <height>480</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="setImgNull">
            <property name="text">
             <string>显示区重置</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PaterRec.qrc"/>
 </resources>
 <connections/>
</ui>
